### PR TITLE
[ci-search] [OCPCRT-303] migrate ci search, define redirector 

### PR DESCRIPTION
**Pull request by ashwindasr**
CI search resources have been migrated and deployed to CR cluster. This PR defines a new redirector to the new place and removes all other resources from the build cluster.

### CODE REVIEW

**Comment by ashwindasr** [comment]
/hold

Until https://github.com/openshift/continuous-release-jobs/pull/1395 is merged

File: clusters/build-clusters/01_cluster/ci-search/deploy.yaml [hunk 1]
```diff
@@ -4,164 +4,70 @@ metadata:
...
 metadata:
-  name: search
-  namespace: ci-search
-spec:
-  selector:
-    app: search
-  ports:
-  - port: 80
-    targetPort: 8080
+  name: ci-search-redirect-config
```
Comment (bradmwilliams) [inline clusters/build-clusters/01_cluster/ci-search/deploy.yaml:25]: nit... Can you move these up under `apiVersion`?

Comment (ashwindasr) [reply clusters/build-clusters/01_cluster/ci-search/deploy.yaml]: added in https://github.com/openshift/release/pull/49035/commits/5e4f750c8426837cf7faa0ebf254941c0262beff

File: clusters/build-clusters/01_cluster/ci-search/deploy.yaml [hunk 2]
```diff
@@ -4,164 +4,72 @@ metadata:
...
-  name: search
+  labels:
+    app: ci-search-redirect
+  name: ci-search-redirect
   namespace: ci-search
-  annotations:
-    image.openshift.io/triggers: '[{"from":{"kind":"ImageStreamTag","name":"ci-search:latest"},"fieldPath":"spec.template.spec.containers[?(@.name==\"web\")].image"}]'
 spec:
-  replicas: 2
+  replicas: 1
```
Comment (bradmwilliams) [inline clusters/build-clusters/01_cluster/ci-search/deploy.yaml:49]: This should probably be `2` to field as many requests as possible

Comment (ashwindasr) [reply clusters/build-clusters/01_cluster/ci-search/deploy.yaml]: added in https://github.com/openshift/release/pull/49035/commits/7ef14e5d1c293c7e05b8bdfe2460684964e42efb

**Review by bradmwilliams — APPROVED** [review]

**Comment by bradmwilliams** [comment]
/hold
Verifying we've got everything in `openshift/release` pointing to the new location

**Review by bradmwilliams — CHANGES_REQUESTED** [review]

**Review by bradmwilliams — APPROVED** [review]

**Review by jmguzik — APPROVED** [review]

**Pull request merged**